web/crux-ui/src/models/container.ts

```diff
@@ -347,6 +355,18 @@ const mergeMarker = (image: Marker, instance: Marker): Marker => {
   }
 }
 
+const mergeMetrics = (image: Metrics, instance: Metrics): Metrics => {
+  if (!instance) {
+    return image?.enabled ? image : null
+  }
+
+  if (!image) {
```

Comment:
if (!image) check since it's not necessary. The first condition handle the case.

Reply:
This checks if there's no instance and image metric config. The first condition only handles if there's no instance metric config.

Reply:
Why return null when there are instance metrics?

Reply:
True, removed the extra check.